### PR TITLE
Perception: Added semantic color codes; Changed default camera parame…

### PR DESCRIPTION
**Pull request by techoe**
…ters of front_6mm

Object: 10 colors
Left lane lines: blue colors
Right lane lines: red colors
Default extrinsic of front_6mm is changed.

### CODE REVIEW

File: modules/perception/camera/tools/offline/visualizer.cc [hunk 1]
```diff
@@ -801,17 +823,17 @@ bool Visualizer::DrawTrajectories(
...
-    cv::Scalar(127, 127, 127));
+    apollo::perception::gray_color);
 
   for (size_t i = 1; i < count; i++) {
     Eigen::Vector4f end_point;
     end_point << static_cast<float>(object->drops[i](0)),
                  static_cast<float>(object->drops[i](1)), 0, 1;
     cv::circle(world_image_, world_point_to_bigimg(end_point), 3,
-      cv::Scalar(127, 127, 127));
+      apollo::perception::gray_color);
```
Comment (Capri2014) [inline modules/perception/camera/tools/offline/visualizer.cc]: gray_color is defined as `cv::Scalar gray_color = cv::Scalar(128, 128, 128);` in your previous definition.

File: modules/perception/camera/tools/offline/visualizer.cc [hunk 2]
```diff
@@ -801,17 +823,17 @@ bool Visualizer::DrawTrajectories(
...
     cv::line(world_image_, world_point_to_bigimg(start_point),
              world_point_to_bigimg(end_point),
-             cv::Scalar(127, 127, 127), trajectory_line_thickness_);
+             apollo::perception::gray_color, trajectory_line_thickness_);
```
Comment (Capri2014) [inline modules/perception/camera/tools/offline/visualizer.cc]: same here.

File: modules/perception/camera/tools/offline/visualizer.cc (hunk 1 above)
Comment (techoe) [reply modules/perception/camera/tools/offline/visualizer.cc]: Yes. 127 was incorrect. (128, 128, 128) is actual gray but human cannot distinguish it.

File: modules/perception/camera/tools/offline/colormap.h [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#pragma once
+
+#include <opencv2/opencv.hpp>
+#include <vector>
```
[the remark this answers is not in the document]
Comment (techoe) [reply modules/perception/camera/tools/offline/colormap.h]: That's true. I changed vector to simple assignment but forgot to remove the include, will remove it.

**Pull request merged**